### PR TITLE
Initializing electron and react project

### PR DESCRIPTION
**Pull request by ihoegen**
This commit initializes and electron project by bringing in all relevant
react files, package json files, and main js files required for the
project to run. It also provides the necesarry babel js file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "translator-core",
```
Comment (vleong2332) [inline package.json]: I think this should be `translation-core`

**Comment by ihoegen** [comment]
Alright I'll edit that

File: package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WycliffeAssociates/8woc.git"
+  },
+  "keywords": [
+    "Translation",
+    "core",
+    "WYcliffe"
```
Comment (rbnswartz) [inline package.json]: Hey look I'm copied on this too. Should probably be Wycliffe not WYcliffe.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    <script src="js/react-dom.js"></script>
+    <script src="js/browser.min.js"></script>
+    <script src='js/sql.js'></script>
+    <script>
+        //Create the database
+        try {
+		var fs = require("fs");
+        var filebuffer = fs.readFileSync('userdata.sqlite');
+        var db = new SQL.Database(filebuffer);
+
```
Comment (vleong2332) [inline index.html]: I believe if we use spaces, these lines will be aligned.

**Pull request merged**